src/newgrf.cpp

```diff
@@ -9780,8 +9785,9 @@ static void FinalisePriceBaseMultipliers()
 	std::vector<int> grf_overrides(num_grfs, -1);
 	for (int i = 0; i < num_grfs; i++) {
 		GRFFile *source = _grf_files[i];
-		uint32_t override = _grf_id_overrides[source->grfid];
-		if (override == 0) continue;
+		auto it = _grf_id_overrides.find(source->grfid);
+		if (it == std::end(_grf_id_overrides)) continue;
+		uint32_t override = it->second;
```

Comment:
plausibly out of scope, but is continuing to use the variable name of `override` a good idea? I realise it's a context-specific keyword, but it highlights weirdly...